### PR TITLE
Add function to get events for any place 

### PR DESCRIPTION
**Pull request by anshg1214**
Add function `get_event_for_place`

### CODE REVIEW

File: brainzutils/musicbrainz_db/event.py [hunk 1]
```diff
@@ -93,3 +94,57 @@ def fetch_multiple_events(mbids, includes=None):
...
+    # map event types to their case sensitive name in musicbrainz.event_type table in the database
+    event_types_mapping = {
+        'award ceremony': 'Award ceremony',
+        'concert': 'Concert',
+        'convention/expo': 'Convention/Expo',
+        'festival': 'Festival',
+        'launch event': 'Launch event',
+        'masterclass/clinic': 'Masterclass/Clinic',
+        'stage performance': 'Stage performance',
+    }
```
Comment (alastair) [inline brainzutils/musicbrainz_db/event.py:132]: As in https://github.com/metabrainz/brainzutils-python/pull/86/files#r862862233, get this list of types fromthe database

**Review by alastair — COMMENTED** [review]
A few comments to clean this up in the same way as #86, but otherwise it looks pretty good, thanks!

File: brainzutils/musicbrainz_db/event.py (hunk 1 above)
Comment (anshg1214) [reply brainzutils/musicbrainz_db/event.py]: Done

File: brainzutils/musicbrainz_db/event.py [hunk 2]
```diff
@@ -93,3 +94,57 @@ def fetch_multiple_events(mbids, includes=None):
...
+
+    place_id = str(place_id)
+    if event_types is None:
+        event_types = []
+    event_types = [event_type.lower() for event_type in event_types]
+
+    includeNULL = False
+    if 'none' in event_types:
+        includeNULL = True
+        event_types.remove('none')
```
[the remark this answers is not in the document]
Comment (anshg1214) [reply brainzutils/musicbrainz_db/event.py]: As we had a bit of a discussion about this in IRC, we don't need all the tabs. We need a tab for Concert, Festival and other. We'll have a parameter `include_events_with_no_type`, which would have a default value of `False`. When the parameter is `True`, the function will return events with the type passed and the events with NULL type.

File: brainzutils/musicbrainz_db/tests/test_event.py [hunk 3]
```diff
@@ -61,3 +61,20 @@ def test_fetch_multiple_events_empty(self, engine):
         ],
             includes=['artist-rels', 'place-rels', 'series-rels', 'url-rels', 'release-group-rels'])
         assert list(events.keys()) == ['d4921d43-bf92-464e-aef4-bba8540fc5bd']
+
+    def test_fetch_get_event_for_place(self, engine):
```
[the remark this answers is not in the document]
Comment (anshg1214) [reply brainzutils/musicbrainz_db/tests/test_event.py:65]: Done

File: brainzutils/musicbrainz_db/event.py [hunk 4]
```diff
@@ -93,3 +94,57 @@ def fetch_multiple_events(mbids, includes=None):
...
 
         return {str(mbid): serialize_events(event, includes_data[event.id]) for mbid, event in events.items()}
+
+
+def get_event_for_place(place_id, event_types=None, limit=None, offset=None):
+    """Get all events linked to a place.
+
+    Args:
+        place_id (uuid): MBID of the place.
+        event_types (list): List of types of events to be fetched.
```
[the remark this answers is not in the document]
Comment (anshg1214) [reply brainzutils/musicbrainz_db/event.py]: Done

**Pull request merged**